install.sh

```diff
@@ -3,9 +3,11 @@
 ###### AUTOMATED INSTALL AND UPDATE SCRIPT ######
 #################################################
 # Written by yomgui1 & Frix_x
-# @version: 1.2
+# @version: 1.3
 
 # CHANGELOG:
+#   v1.3: added a warning on first install to be sure the user wants to install klippain and fixed a bug
+#         where some artefacts of the old user config where still present after the install (harmless bug but not clean)
```

Comment:
is the info regarding gcode shell missing?

Reply:
This is fixed in the last commit